src/_data/team/serban-costin.json

```diff
@@ -0,0 +1,12 @@
+{
+    "order": 16,
+    "name": "Serban Costin",
+    "title": "Front End Developer",
+    "github": "cstns",
+    "linkedin": "serban-costin-32a07894",
+    "country": "RO",
+    "headshot": "serban.png",
```

Comment:
This file is missing :)

Reply:
yes, I have an [art request](https://github.com/FlowFuse/website/issues/1947) for the photo and still need to come up with something for the description. I marked the PR as a draft until I get these sorted